validator/sawtooth_validator/journal/consensus/poet/poet_consensus/poet_transaction_block.py

```diff
@@ -237,6 +238,7 @@ def is_valid(self, journal):
 
             try:
                 self.wait_certificate.check_valid(
+                    poet_enclave_module=None,
```

Comment:
Not clear why this is set to `None`. It looks like downstream `_enclave_wait_certificate(self, poet_enclave_module)` will try to call using the passed enclave. 
`poet_enclave_module.deserialize_wait_certificate(...)`